source/administration/changelog.md

```diff
@@ -54,55 +48,75 @@ Release Date 2019-04-16
  - Fixed an issue where the text box reverted to default size after a user returned from the Integrations page.
  - Fixed an issue where the profile popover wasn't allowed to close itself when opened through an at-mention.
  - Fixed an issue where filtering by first name with Korean characters no longer worked for at-mentions.
+ - Fixed an issue where the **Remove MFA** option was visible for all users when **Enforce MFA** was enabled.
  
 ### Compatibility
 
- - Deprecated configurable ``timezones.json`` in favour of the existing hard-coded list built into the server. **XXXX NEEDS MORE DETAILS**
+#### Deprecated Features
+
+ - Deprecated configurable ``timezones.json`` in favour of the existing hard-coded list built into the server.
 
-### config.json
+#### config.json
 
 Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
  
 #### Changes to Team Edition and Enterprise Edition:
 
 - Under ``"ExperimentalSettings":`` in ``config.json``:
-   - Added ``"RestrictSystemAdmin": false``, to 
+   - Added ``"RestrictSystemAdmin": false``, to optionally constrain even system admins from changing critical settings.
 - Under ``"ServiceSettings":`` in ``config.json``:
    - Added ``"MinimumHashtagLength": 3``, to add the ability to change the minimum length of hashtags.
-- Under ``"ElasticsearchSettings":`` in ``config.json``: 
-   - Added ``"EnableAutocomplete": false``, ``"ChannelIndexReplicas": 1``, ``"ChannelIndexShards": 1``, ``"UserIndexReplicas": 1``, and ``"UserIndexShards": 1``, to enable autocompletion for Elasticsearch queries.
 
 ### API Changes
  - Added ``GetUsers`` API method to add the ability to list users.
  - Added the ``SearchPostsInTeam`` method to the plugin API to add the ability to search posts in a team.
  - Added ``GetTeamMembersForUser`` and ``GetChannelMembersForUser`` to the plugin API to add the ability to get team and channel members for a specific user.
  - Added ``GetBundleInfo() string`` method to the plugin API to add the ability to store assets elsewhere.
- 
-### Websocket Event Changes
 
 ### Database Changes
+ - Granted the following permissions for the System Admin, in preparation for an upcoming bot accounts feature:
+   - PERMISSION_CREATE_BOT
+   - PERMISSION_READ_BOTS
+   - PERMISSION_READ_OTHERS_BOTS
+   - PERMISSION_MANAGE_BOTS
+   - PERMISSION_MANAGE_OTHERS_BOTS
 
 ### Known Issues
+ - On a server using a subpath, the URL opens a blank page if the system admin changes the Site URL in the System Console UI. The system admin should restart the server to fix it.
+ - Login does not work when Custom Terms of Service is enabled and MFA is enforced.
+ - Google login fails on the Classic mobile apps.
+ - User can receive a video call from another browser tab while already on a call.
```

Comment:
To what is this referring?

Reply:
Specifically:
> - User can receive a video call from another browser tab while already on a call.

Reply:
I wasn't able to find a Jira ticket so I removed it from the list for now,

Reply:
PS: This was an issue with the legacy WebRTC calls